### PR TITLE
chore(node_compat): add deno task for setting up and running tests

### PR DESCRIPTION
**Pull request by wafuwafu13**
- `deno task setup`: setting up tests
- `deno task test`: running specific tests

### CODE REVIEW

File: tools/node_compat/deno.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "tasks": {
+    "setup": "deno run --allow-read --allow-write ./setup.ts",
+    "test": "deno test -A ../../cli/tests/node_compat/test.ts -- $TARGET"
```
Comment (dsherret) [inline tools/node_compat/deno.json:4]: Why have a variable? Couldn't someone just do `deno task test buffer`?

Comment (wafuwafu13) [reply tools/node_compat/deno.json]: It becomes simple and fix it. https://github.com/denoland/deno/pull/19293/commits/d2d0ca9fa4569199557409cb1bd1782cb6a4a06b

File: tools/node_compat/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Node.js compat testing in Deno repository.
    example, if you want to add `test/paralles/test-foo.js` from Node.js test
    cases, then add `test-foo.js` entry in `tests.parallel` array property in
    `config.jsonc`
-1. Run `./tools/node_compat/setup.ts`
+1. Run `deno task setup`
```
[the remark this answers is not in the document]
Comment (kt3k) [reply tools/node_compat/README.md:26]: Ok. Then let's just note that cwd needs to be changed.

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**